consts/others.go

```diff
@@ -1,11 +1,12 @@
 package consts
 
 const (
-	StrBasicAuthPrefix              string = "Basic "
-	StrTokenAuthPrefix              string = "Auth Token "
-	StrEmailTokenVerificationPrefix string = "Email Token "
-	StrBasicAuthAttempt             string = "Basic Auth Attempt"
-	StrTokenAuthAttempt             string = "Token Auth Attempt"
-	StrMdBasicAuthHeader            string = "authorization"
-	StrMdAuthToken                  string = "auth_token"
+	StrBasicAuthPrefix               string = "Basic "
+	StrTokenAuthPrefix               string = "Auth Token "
+	StrEmailTokenVerificationPrefix  string = "Email Token "
+	StrBasicAuthAttempt              string = "Basic Auth Attempt"
+	StrAuthTokenAttempt              string = "Auth Token Attempt"
+	StrEmailTokenVerificationAttempt string = "Email Token Verification Attempt"
+	StrMdBasicAuthHeader             string = "authorization"
```

Comment:
Should it be capital "Authorization" for consistency with the previous ones

Reply:
Good catch Mandy. The convention is upper case.
https://en.wikipedia.org/wiki/Basic_access_authentication